### PR TITLE
list.sorted compare rule changed

### PR DESCRIPTION
**Pull request by EugeneDar**

### CODE REVIEW

**Comment by EugeneDar** [comment]
@yegor256 please check.

File: paper.tex [hunk 1]
```diff
@@ -498,7 +498,7 @@ \subsection{List}
 
 The attribute \deff{slice} is a part of the array.
 
-The attribute \deff{sorted} is a new array with elements sorted using given comparison abstract object with two free attributes.
+The attribute \deff{sorted} is a new array with elements sorted using the .lt method of the elements passed.
```
Comment (yegor256) [inline paper.tex:501]: @EugeneDar it's better to do say:

```
using \ff{lt} attribute of the elements passed
```

it's more in line with our terminology

**Comment by yegor256** [comment]
@EugeneDar see above pls

File: paper.tex (hunk 1 above)
Comment (EugeneDar) [reply paper.tex]: @yegor256 I'll fix it.

**Comment by EugeneDar** [comment]
@yegor256 please merge.

**Comment by yegor256** [comment]
@EugeneDar thanks!

**Pull request merged**